src/tests/enricher/test_enrich_bag.py

```diff
@@ -25,9 +40,39 @@ def test_enrich_nummeraanduidingen(self):
         self.assertEqual(entities[0]['ligt_in_bag_woonplaats'],'3594')
         self.assertEqual(entities[1]['ligt_in_bag_woonplaats'],'1024')
 
+        self.assertEqual(entities[0]['amsterdamse_sleutel'],'sleutel')
+        self.assertEqual(entities[1]['amsterdamse_sleutel'],'sleutel2')
+
```

Comment:
What happens if the key cannot be found? Can you add a test for this.

Reply:
Attribute should be empty, added in the test